array-filter/filter.js

```diff
@@ -0,0 +1,10 @@
+function filter(array, predicate) {
+  const returnArray = [];
+  for (let i = 0; i < array.length; i++) {
+    const resultValue = predicate(array[i], i, array);

```

Comment:
Nice job adding all the arguments to match the original functionality of filter!